### ISSUE TITLE
Display Persistent Storage Resource Info in Provider Details

### ISSUE THREAD

**Issue by anilmurty** [akash-network/console#245]
Provider details page currently only shows ephemeral storage and not persistent storage. For example https://console.akash.network/providers/akash1rfqs6aajq2d4azhjj88wav2d6ra6vvuzn58dt5?network=mainnet has 1, 1.7TB drives with persistent storage but that info isn't displayed

![image](https://github.com/akash-network/cloudmos/assets/19495789/a9506ad8-c299-45ed-b15b-05d986a251e5)

```
root@node1:~# provider-services status akash1rfqs6aajq2d4azhjj88wav2d6ra6vvuzn58dt5 --node [https://akash-rpc.polkachu.com:443](https://akash-rpc.polkachu.com/)
{
  "cluster": {
    "leases": 5,
    "inventory": {
      "active": [
        {
          "cpu": 100,
          "gpu": 0,
          "memory": 100663296,
          "storage_ephemeral": 6291456
        },
        {
          "cpu": 2000,
          "gpu": 0,
          "memory": 4294967296,
          "storage_ephemeral": 2147483648,
          "storage": {
            "beta3": 5368709120
          }
        },
        {
          "cpu": 1000,
          "gpu": 0,
          "memory": 536870912,
          "storage_ephemeral": 536870912
        },
        {
          "cpu": 2000,
          "gpu": 0,
          "memory": 4294967296,
          "storage_ephemeral": 4294967296,
          "storage": {
            "beta3": 10737418240
          }
        },
        {
          "cpu": 1000,
          "gpu": 2,
          "memory": 17179869184,
          "storage_ephemeral": 53687091200
        }
      ],
      "pending": [
        {
          "cpu": 2000,
          "gpu": 0,
          "memory": 4294967296,
          "storage_ephemeral": 4294967296
        }
      ],
      "available": {
        "nodes": [
          {
            "name": "ubuntu-63-222-125-248",
            "allocatable": {
              "cpu": 56000,
              "gpu": 8,
              "memory": 540904771584,
              "storage_ephemeral": 99834798412
            },
            "available": {
              "cpu": 44000,
              "gpu": 6,
              "memory": 505532108800,
              "storage_ephemeral": 34867126604
            }
          }
        ],
        "storage": [
          {
            "class": "beta3",
            "size": 1195915214848
          }
        ]
      }
    }
  },
  "bidengine": {
    "orders": 1
  },
  "manifest": {
    "deployments": 2
  },
  "cluster_public_hostname": "provider.cato.akash.pub"
}
```

**Comment by Redm4x** [comment]
Update June 25, 2024: 

This is not just a display issue. I'm working on improving the indexing part to distinguish between ephemeral & persistent storage and different storage classes. Once done we will be able to add persistent storage stats in the provider page and other usage graphs.

**Comment by Redm4x** [comment]
Update July 16, 2024:

This is fixed in #267, it will be deployed in the coming days. Persistent and ephemeral storage are combined in the same graphs for now, but could be split in their own graphs/tiles in the future.

**Comment by Redm4x** [comment]
Update July 23, 2024:

This was deployed this morning, reported global storage capacity went from 501TB to 609TB on stats.akash.network. Cato provider now correctly report 1.66TB of total storage.
![image](https://github.com/user-attachments/assets/665e2c4b-fff3-4829-934c-e99c9fd5a90a)
